src/test/java/org/example/integration/AuthIntegrationTest.java

```diff
@@ -0,0 +1,85 @@
+package org.example.integration;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.example.JobRoleManagerApplication;
+import org.example.JobRoleManagerConfiguration;
+import org.example.models.LoginRequest;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Base64;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthIntegrationTest {
+    private final String apiUrl;
+
+    private static final DropwizardAppExtension<JobRoleManagerConfiguration> APP =
+            new DropwizardAppExtension<>(JobRoleManagerApplication.class);
+
+    public AuthIntegrationTest() {
+        String apiUrl = System.getenv("API_URL");
+        if (apiUrl == null || apiUrl.isEmpty()) {
+            this.apiUrl = "http://localhost:8080/api/";
+        } else {
+            this.apiUrl = apiUrl;
+        }
+    }
+    // FIX ME: Remove disabled annotation when API is deployed so test can run w/ correct URL
```

Comment:
Just commenting so we have a shared understand (this doesn't need to be fixed now): we should make sure our GitHub action runs integration tests after deployment

Reply:
Yup thats correct